src/domains/citizen-auth-common/06_data.tf

```diff
@@ -33,3 +33,18 @@ data "azurerm_linux_function_app" "lollipop_function" {
   resource_group_name = data.azurerm_resource_group.lollipop_function_rg.name
 }
 #######################
+
+data "azurerm_resource_group" "data_rg_itn" {
+  name = "${local.project_itn}-data-rg-01"
+}
+
+data "azurerm_virtual_network" "vnet_itn" {
+  name                = "${local.prefix}-${local.env_short}-itn-common-vnet-01"
+  resource_group_name = "${local.prefix}-${local.env_short}-itn-common-rg-01"
+}
+
+data "azurerm_subnet" "subnet_pep_itn" {
+  name                 = "io-p-itn-pep-snet-01 "
+  resource_group_name  = data.azurerm_virtual_network.vnet_itn.resource_group_name
+  virtual_network_name = data.azurerm_virtual_network.vnet_itn.name
+}
```

Comment:
We can use the common_values module to get the information of the PEPs subnet.
I'd add the following code to the main.tf file:

```
module "common_values" {
 source = "github.com/pagopa/io-infra//src/_modules/common_values?ref=main"
}
```

```suggestion
```